lesson-2-working-with-state/README.md

```diff
@@ -0,0 +1,23 @@
+# Lesson 8.2 - Working with State
+
+### Key Terms:
+
++ state
++ setState
++ `useState` hook
++ reducer
+
+### Essential Questions:
+
```

Comment:
They are coming off of `useState`. They may be thinking "why do I need this hook? Can't I just `useState`? I will add a question here comparing and contrasting the two.